esm/interface/custom-element-registry.js

```diff
@@ -190,4 +190,16 @@ export class CustomElementRegistry {
     const info = this.registry.get(localName);
     return info && info.Class;
   }
+
+  /**
+   * @param {Function} Class **Class** of custom element
+   * @returns {string?} found tag name or null
+   */
+    getName(Class){
```

Comment:
some indentation issue here ... one less space for `getName` and the rest of the code, otherwise this looks good to me.

Reply:
Should be fixed now!

Thanks for your fast response.